portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,15 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+const navigate = (pageid) => document.getElementById(pageid).scrollIntoView({behavior:"smooth",block:"start"});
+
+const initialiseNavigation = () => {
+  var navigationItems = document.getElementsByClassName("navigation-item");
```

Comment:
[don't use `var`](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)

Reply:
Changed from var to const with commit 7a83ce2